python/lsst/meas/extensions/shapeHSM/_hsm_shape.py

```diff
@@ -41,6 +41,17 @@
 ]
 
 
+def inherit_prop_docs(cls):
+    """Decorator to inherit docstrings for properties from base classes."""
+    for name in dir(cls):
+        attr = getattr(cls, name, None)
+        if isinstance(attr, property):
+            base_prop = getattr(cls.__bases__[0], name, None)
+            if isinstance(base_prop, property) and attr.__doc__ is None:
```

Comment:
Do we need a second `isinstance` check here?

Reply:
No, I used a different approach in my last commit.